include/sys/dmu_objset.h

```diff
@@ -197,6 +197,9 @@ struct objset {
 	dmu_objset_upgrade_cb_t os_upgrade_cb;
 	boolean_t os_upgrade_exit;
 	int os_upgrade_status;
+
+	/* prefetch */
```

Comment:
This comment and blank line do not worth screen space they take.

Reply:
I inserted blank+comment to separate this new item from the above items+comment (which reads "kernel thread to upgrade this dataset"). Removing the blank+comment will let `os_prefetch` to be "included" in the previous item groups and comment. Can be this an issue? Would it be better to move `os_prefetch` just after `os_recordsize` (given that block comment)?